Apps/PackageTest/android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.0.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.2-all.zip
```

Comment:
If you upgrade this to 6.3 we can use JDK 14

Reply:
Updated, hopefully the lab build succeeds. 🤞🏻